draft-ietf-httpbis-http2bis.xml

```diff
@@ -467,9 +471,9 @@ HTTP Frame {
                 An 8-bit field reserved for boolean flags specific to the frame type.
             </t>
             <t>
-                Flags are assigned semantics specific to the indicated frame type.  Unused flags are those that have
-                no defined semantics for a particular frame type, and MUST be ignored and MUST be left
-                unset (0x0) when sending.
+              Flags are assigned semantics specific to the indicated frame type.  Unused flags are
+              those that have no defined semantics for a particular frame type. Unused flags MUST be
+              ignored on receipt and MUST be left unset (0x0) when sending.
```

Comment:
Was the decent here intentional?

Reply:
The indent was poor before, but I "fixed" it when reflowing the text.